src/client/sandbox/code-instrumentation/properties/index.js

```diff
@@ -274,16 +274,18 @@ export default class PropertyAccessorsInstrumentation extends SandboxBase {
                     var isStyleEl  = domUtils.isStyleElement(el);
                     var isScriptEl = domUtils.isScriptElement(el);
 
-                    if (value) {
+                    var processedValue = value !== null && value !== void 0 ? String(value) : value;
```

Comment:
I think it's not quite right check. When I try to set `null` to `innerHTML`:
```text
document.body.innerHTML = null
null
document.body.innerHTML
""
```
But if I try to set `undefined`:
```text
document.body.innerHTML = void 0
undefined
document.body.innerHTML
"undefined"
```
We should take this into account.